content/admin/org_settings/integrations/MicrosoftTeams.md

```diff
@@ -0,0 +1,33 @@
+<!--
+title: "MicrosoftTeams Integration"
+description: "Integrating MicrosoftTeams with Contrast"
+tags: "Admin organization settings integrations microsoftTeams"
+-->
+
+Use Contrast's Microsoft Teams integration to receive notifications from Contrast in your configured Microsoft Teams instance.
+
+## Setup
+
+### Microsoft Teams
+
```

Comment:
It looks like our Integrations pages differ in the way they present info about external resources (some provide links, some provide prereqs, some write out the steps). In the future, Docs can develop guidance/templates for this. I think for now, this is OK.